<launch>

    <!-- Robot namespace -->
    <arg name="robot_name" default="my_gen3_lite" />
    <arg name="start_delay_seconds" default="0" />

    <!-- Launch the example in that namespace -->
    <node name="grasp_and_grip" pkg="kortex_examples" type="grasp_and_grip"
        output="screen" launch-prefix="bash -c 'sleep $(arg start_delay_seconds); $0 $@' ">
        <param name="robot_name" value="$(arg robot_name)" />
    </node>

</launch>